src/locales/uk/translation.json

```
@@ -85,6 +85,8 @@
   "division": "Розділ",
   "downloadContract": "Завантажити договір",
   "downloadPEPDatabase": "Завантаження всієї бази РЕР",
+  "downloadXlsx": "Зараз буде вигрузка даних в форматі .xlsx",
+  "downloadXlsxConfirmation": "Після завершення на Ваш email буде відправлено лист з посиланням на скачування цього файлу. Продовжити?",
```

put **all** messages what have special values to other object:
```suggestion
  "export": {
    "downloadXlsx": "Зараз буде вигрузка даних в форматі .xlsx",
    "downloadXlsxConfirmation": "Після завершення на Ваш email буде відправлено лист з посиланням на скачування цього файлу. Продовжити?",
  },
```

---

done